src/StackAdmin/Dns/Dns.sln_

```diff
@@ -0,0 +1,106 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Should the name of this file have an underscore at the end?

Reply:
removed this file